definitions/checks/foreman_tasks/not_running.rb

```diff
@@ -9,8 +9,17 @@ class NotRunning < ForemanMaintain::Check
     end
 
     def run
-      assert(feature(:foreman_tasks).running_tasks_count == 0,
-             'There are actively running tasks in the system')
+      task_count = feature(:foreman_tasks).running_tasks_count
+      assert(task_count == 0,
+             failure_message(task_count),
+             :next_steps =>
+               [Procedures::ForemanTasks::Resume.new,
```

Comment:
The resume is not valid for `running` tasks. We could have a procedure, that polls and waits for the tasks to finish.

Reply:
In the running_tasks_count, I have mentioned  filter state field  should be either running or paused. That's why I mentioned this step.

Should I remove paused from state filter values?

Reply:
Also, please correct me if I am misunderstood.
Here, polls and wait for tasks to finish means we need to ask user to enter timeframe to wait.
After timeframe elapsed, we need to exit from procedure.

Reply:
@iNecas ,

I have queries regarding this:
- Do we need to set any timeout for this procedure to exit? Or we really need to wait till all tasks to finish.
- What interval should we set before retrying status of tasks?

Reply:
We have the `ForemanTasks::NotPaused` check for the paused, so this should be for running only.

Having a timeout is a good idea. I would suggest something around 5 minutes for start.

For the retry interval, I would go for every 10 seconds, as it's not very expensive operation.

Reply:
@iNecas ,
Is it okay if I change order for NotRunning check as it is showing after NotPaused check?

I know that it is a rare situation where any running task which will get paused somehow in NotRunning check. But in that scenario it will proceed with upgrade.

I have pushed the changes but they are incomplete. I just saw your comments.
I will update PR.